### ISSUE TITLE
Webhook Thread Support

### ISSUE THREAD

**Issue by AtoraSuunva** [DragonPlayerX/ScreenshotManager#8]
Webhooks can send messages to threads by using a `thread_id` [query parameter](https://discord.com/developers/docs/topics/threads#webhooks), although trying to add one causes an error to occur when trying to upload ("The given Webhook URL is invalid." [here](https://github.com/DragonPlayerX/ScreenshotManager/blob/master/ScreenshotManager/Core/MenuManager.cs#L642))

```ini
# Fails
WebhookURL = https://canary.discord.com/api/webhooks/<id>/<token>?thread_id=<thread>
# Identical webhook without `thread_id`, works
WebhookURL = https://canary.discord.com/api/webhooks/<id>/<token>
```

Quickly looking through the code, it looks like the `=` from the query parameter causes it to fail to parse the line (since it splits into 3 parts, while it expects 2), making it silently fail without storing the webhook url

https://github.com/DragonPlayerX/ScreenshotManager/blob/master/ScreenshotManager/Config/DiscordWebhookConfiguration.cs#L75

Something like a `ThreadID` config parameter could work, I think

**Comment by DragonPlayerX** [comment]
I guess, instead of making a new config parameter, it would be enough to improve the parsing method to respect multiple "=" symbols.

**Comment by DragonPlayerX** [comment]
The parsing for webhook query parameters was fixed in commit 4434ae8cdef2760479dd038e539baf1f4b3287d3 and included in release [Screenshot Manager 2.3.2](https://github.com/DragonPlayerX/ScreenshotManager/releases/tag/2.3.2).
